platform-includes/distributed-tracing/custom-instrumentation/server/javascript.mdx

```diff
@@ -0,0 +1,100 @@
+Trace propagation is set up automatically in this SDK. By default, outgoing HTTP requests will automatically be instrumented for you.
+For other cases where you may want to continue traces (for example, in a job queue or when working with web sockets), you can manually extract and inject tracing information.
+
+If you want to use automated distributed tracing but not tracing, set the `tracesSampleRate` option to `0`.
```

Comment:
I know this is the same as the other page where I called out this comment, but this needs to be explained better here, too.